test/WebJobs.Script.Tests/TestScripts/Node/EventHubSender/function.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "bindings": {
+    "input": [
+      {
+        "type": "manualTrigger"
+      }
+    ],
+    "output": [
+      {
+        "type": "eventHub",
+        "name" :  "output",
+        "ConnectionString": "%AzureWebJobsEventHubSender%",
```

Comment:
these connection/path strings are different than those in the sample (e.g. EHSender). Is that intentional? Hopefully we don't need to add 2 sets of env settings on the CI machine